great_expectations/dataset/sqlalchemy_dataset.py

```diff
@@ -249,7 +249,7 @@ def __init__(self, table_name=None, engine=None, connection_string=None,
             #was used for a temp table and raising an error
             schema = None
             # dashes are special characters in most databases so use underscores
-            table_name = "ge_tmp_" + str(uuid.uuid4()).replace("-", "_")
+            table_name = "ge_tmp_" + str(uuid.uuid4())[:8]
```

Comment:
nitpick: please remove the obsolete comment about underscores.

Reply:
✔️